tensorboard/components/vz_line_chart2/line-chart.ts

```diff
@@ -759,12 +767,25 @@ export class LineChart {
   }
 
   /**
-   * Set the data of a series on the chart.
+   * Sets the data of a series on the chart.
    */
-  public setSeriesData(name: string, data:  vz_chart_helpers.ScalarDatum[]) {
+  public setSeriesData(name: string, data: vz_chart_helpers.ScalarDatum[]) {
     this.getDataset(name).data(data);
   }
 
+  /**
+   * Sets getTitle callback on a series.
+   */
+  public setSeriesTitleGetter(name: string, getTitle: (name: string) => string) {
```

Comment:
As discussed, maybe a little more consistent w/ existing code to make the experiment a field on the dataset metadata and then have a "titleGetter" bound to the chart as a whole, vs binding individual closures onto each series.